src/WebJobs.Script/Sanitizer.cs

```diff
@@ -16,7 +16,7 @@ internal static class Sanitizer
 
         // List of keywords that should not be replaced with [Hidden Credential]
         private static readonly string[] AllowedTokens = new string[] { "PublicKeyToken=" };
-        internal static readonly string[] CredentialTokens = new string[] { "Token=", "DefaultEndpointsProtocol=http", "AccountKey=", "Data Source=", "Server=", "Password=", "pwd=", "&amp;sig=", "&sig=", "SharedAccessKey=" };
+        internal static readonly string[] CredentialTokens = new string[] { "Token=", "DefaultEndpointsProtocol=http", "AccountKey=", "Data Source=", "Server=", "Password=", "pwd=", "&amp;sig=", "&sig=", "?sig=", "SharedAccessKey=" };
```

Comment:
Aren't there existing tests that can also be updated as in v4?

Reply:
Hi @mathewc 
It looks removed on v3.x branch. I don't know why. :( So that I followed.